components/MaskedTransition/src/private/MDCMaskedTransitionMotionSpec.h

```diff
@@ -17,26 +17,31 @@
 #import <Foundation/Foundation.h>
 #import <MotionInterchange/MotionInterchange.h>
 
-struct MDCMaskedTransitionMotionTiming {
+typedef struct MDCMaskedTransitionMotionTiming {
   MDMMotionTiming iconFade;
   MDMMotionTiming contentFade;
   MDMMotionTiming floodBackgroundColor;
   MDMMotionTiming maskTransformation;
   MDMMotionTiming horizontalMovement;
   MDMMotionTiming verticalMovement;
   MDMMotionTiming scrimFade;
-};
-typedef struct MDCMaskedTransitionMotionTiming MDCMaskedTransitionMotionTiming;
+} MDCMaskedTransitionMotionTiming;
 
-struct MDCMaskedTransitionMotionSpec {
+typedef struct MDCMaskedTransitionMotionSpecContext {
   MDCMaskedTransitionMotionTiming expansion;
   MDCMaskedTransitionMotionTiming collapse;
   BOOL shouldSlideWhenCollapsed;
   BOOL isCentered;
-};
-typedef struct MDCMaskedTransitionMotionSpec MDCMaskedTransitionMotionSpec;
+} MDCMaskedTransitionMotionSpecContext;
 
-FOUNDATION_EXPORT struct MDCMaskedTransitionMotionSpec fullscreen;
-FOUNDATION_EXPORT struct MDCMaskedTransitionMotionSpec bottomSheet;
-FOUNDATION_EXPORT struct MDCMaskedTransitionMotionSpec bottomCard;
-FOUNDATION_EXPORT struct MDCMaskedTransitionMotionSpec toolbar;
+@interface MDCMaskedTransitionMotionSpec: NSObject
+
++ (MDCMaskedTransitionMotionSpecContext)fullscreen;
++ (MDCMaskedTransitionMotionSpecContext)bottomSheet;
++ (MDCMaskedTransitionMotionSpecContext)bottomCard;
++ (MDCMaskedTransitionMotionSpecContext)toolbar;
```

Comment:
Would this be easier to use in Swift if these were class, nonnull, readonly properties?

Reply:
Yep! Though these are private APIs so they are only used by the corresponding Objective-C class at this time, I'm happy to make sure that if we do expose the specs that the APIs are easier to use in Swift.